bigquery/google/cloud/bigquery/dbapi/_helpers.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import collections
+import datetime
+import numbers
+import six
+import time
+
+from google.cloud import bigquery
+from google.cloud.bigquery.dbapi import exceptions
+
+
+def wait_for_job(job):
+    """Waits for a job to complete by polling until the state is `DONE`.
+
+    Raises a DatabaseError if the job fails.
+
+    :type: :class:`~google.cloud.bigquery.job._AsyncJob`
+    :param job: Wait for this job to finish.
+    """
+    while True:
+        job.reload()
+        if job.state == 'DONE':
+            if job.error_result:
+                raise exceptions.DatabaseError(job.errors)
+            return
+        time.sleep(1)
+
+
+def scalar_to_query_parameter(value, name=None):
+    """Convert a scalar value into a query parameter.
+
+    Note: You must use the unicode type for string parameters in Python 2.
+
+    Raises a :class:`~ google.cloud.bigquery.dbapi.exceptions.ProgrammingError`
```

Comment:
1. Please convert this to a `:raises:` section
1. You have a space between `~` and `google` that doesn't belong